src/App.xaml.cs

```diff
@@ -105,6 +106,9 @@ public App()
             services.AddSingleton<IAppInstallManagerService, AppInstallManagerService>();
             services.AddSingleton<IPackageDeploymentService, PackageDeploymentService>();
             services.AddSingleton<IScreenReaderService, ScreenReaderService>();
+            services.AddSingleton<IComputeSystemService, ComputeSystemService>();
+            services.AddSingleton<IComputeSystemManager, ComputeSystemManager>();
+            services.AddSingleton<ToastNotificationService>();

```

Comment:
@dkbennett do we have toast notifications in Dev Home core already?  I just want to make sure we don't have 2 solutions now.

Reply:
For what it's worth we added this here now because we needed a way to tell the user that they arent in the Hyper-V admin group when they open the environments tool page or the "Setup a target" flow in machine configuration. (You can't do any Hyper-V admin functionality unless you are in an elevated process or you're in the Hyper-v admin group). E.g the Hyper-V manager in the Windows OS actually runs elevated, we just don't see the UAC.

But we have plans to actually create an elevated process to add the user to this group, without the need to show a toast notification (work is to be done before Build), in which case we'd remove this. Though I know there are other ways to tell the user this without the need for the toast notification, it's what we ended up doing for now. So, in the end, this is only temporary. If David already has a way to send notifications, we can use that. Otherwise, just know this is temporary.

Reply:
This is totally fine.  I know David added notifications to the extensions, but I wanted to check if there was existing code in Dev Home to do this.

Reply:
Consider adding an interface for this service for mock-ability